web/modules/custom/par_data/src/Entity/ParDataEntity.php

```
@@ -462,6 +476,16 @@ public function getCompletionPercentage($include_deltas = FALSE) {
     return $total > 0 ? ($completed / $total) * 100 : 0;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function setNewRevision($value = TRUE) {
+    $this->setRevisionCreationTime(REQUEST_TIME);
+    $this->setRevisionAuthorId(\Drupal::currentUser()->id());
+
+    parent::setNewRevision($value);
```

just a sanity check on this line, not sure what this would do, would it fail silently if parent::setNewRevision() isn't implemented?

---

It's always implemented, it's part of the core ContentEntityBase.

---

Was just testing you @kalpaitch